src/main/java/appeng/tile/spatial/SpatialAnchorTileEntity.java

```diff
@@ -65,7 +66,16 @@
 import appeng.util.IConfigManagerHost;
 
 public class SpatialAnchorTileEntity extends AENetworkTileEntity
-        implements IGridTickable, IConfigManagerHost, IConfigurableObject, IOverlayDataSource {
+        implements IGridTickable, IConfigManagerHost, IConfigurableObject, IOverlayDataSource, IMovableTile {
+
+    /**
+     * Loads this radius after being move via a spatial transfer. This accounts for the anchor not being placed in the
+     * center of the SCS, but not as much as trying to fully load a 128 cubic cell with 8x8 chunks. This would need to
+     * load a 17x17 square.
+     * 
+     * TODO: 1.17 Break API compat and pass plot id or plot-descriptor to {@link IMovableTile}.
```

Comment:
Does not have to be breaking. Could be a more advanced version of `IMovableTile` or `IMovableHandler`. Maybe not at tile level, but the whole SCS construct.